src/python/WMCore/MicroService/Unified/MSCore.py

```diff
@@ -110,9 +111,20 @@ def getTransferInfo(self, wname):
 
 
         :param wname: workflow name
-        :return: a tranfer ids document
+        :return: a generator of transfer records
         """
-        return self.reqmgrAux.getTransferInfo(wname)
+        records = self.reqmgrAux.getTransferInfo(wname)
+        # according to https://github.com/dmwm/WMCore/wiki/ReqMgr2-MicroService-Monitor
+        # we may need to select only record since last updated according
+        # to configuration UpdateInterval setting
+        for rec in records:
+            if since:
+                interval = self.msConfig.get('UpdateInterval', 4*24*60*60)
```

Comment:
let's make it every 6h please. And don't forget updating the deployment configuration file.